lib/trento/application/read_models/application_instance_read_model.ex

```diff
@@ -7,6 +7,8 @@ defmodule Trento.ApplicationInstanceReadModel do
 
   import Ecto.Changeset
 
+  require Trento.Domain.Enum.Health, as: Health
```

Comment:
this should be called `Trento.Domain.Enums.Health` since the folder is called enums